cmd/zed/zed.d/statechange-led.sh

```diff
@@ -66,11 +68,11 @@ check_and_set_led()
 	# If another process is accessing the LED when we attempt to update it,
 	# the update will be lost so retry until the LED actually changes or we
 	# timeout.
-	for _ in $(seq 1 5); do
+	for _ in 1 2 3 4 5; do
```

Comment:
What is the point of this change?

Reply:
As noted: seq is a very spotty GNU extension (and also not checked for at the top), so when not available this resolves to `for _ in ; do` which I assume you can guess why is less than optimal. (Plus, it's literally five numbers, do you really need to spawn a process for it?)

Reply:
Has checkbashisms said that?

Reply:
Hm, I've misremembered apparently, according to FreeBSD's seq(1) it's had a tumultuous history (as ported from GNU shellutils and/or P9) and should be available on most recent fully-configured UNIXes, but I've also definitely used systems without it in recent years.
Plus, if the portability to small systems doesn't appeal to you, then the performance gains of not using a process to generate 5 numbers that are, in sum, shorter than the process substitution definitely will.

and dunno, haven't run checkbashisms